Transform.cs

```diff
@@ -132,6 +141,12 @@ internal static Angle _FromQuat(Quaternion quaternion)
             return new Angle(yawDegree: vec.y, pitchDegree: -vec.x);
         }
 
+        public static Angle Look(Displacement d) => _FromQuat(Quaternion.LookRotation(d._AsVec3));
+        public static Angle Lerp(Angle a, Angle b, float t) =>
```

Comment:
same as above. I might be able to replace this with `AdvanceToTarget` or something